Doc/tutorial/controlflow.rst

```diff
@@ -166,7 +166,7 @@ arguments.  In chapter :ref:`tut-structures`, we will discuss in more detail abo
 The :keyword:`break` statement breaks out of the innermost enclosing
 :keyword:`for` or :keyword:`while` loop.
 
-Loop statements may have an :keyword:`!else` clause; it is executed when the loop
+The :keyword:`for` loop statements may have an extra :keyword:`!else` clause; it is executed when the loop
```

Comment:
This just seems bad. You restricted it to `for` statements, but it also applies to `while` statements. And the `else` clause isn't really "extra" (I even checked some dictionaries, they all defined it like *"added to an existing or usual amount or number; additional. Example: they offered him an extra thirty-five cents an hour"*. And loop statements can only have *one* `else` clause.)